Makefile

```diff
@@ -1,11 +1,17 @@
 OPERATOR_SDK_VERSION ?= v0.18.2
-IMAGE_REGISTRY ?= quay.io
+IMAGE_REGISTRY ?= quay.io/oyahud
 OPERATOR_IMAGE ?= kubevirt-ssp-operator
 IMAGE_TAG ?= latest
+OPERATOR_BUILD_ARGS ?= "-v -mod=vendor"
 
 operator-sdk:
 	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
-	chmod 0755 operator-sdk
+	chmod 0755 ./operator-sdk
 
 operator-build:
```

Comment:
This make target uses operator-sdk, so it should depend on the `operator-sdk` target:
``` make
operator-build: operator-sdk
    go mod tidy
    ...
```